plugin/src/main/kotlin/de/triplet/gradle/play/internal/PlayPublishTaskBase.kt

```diff
@@ -72,4 +76,46 @@ abstract class PlayPublishTaskBase : DefaultTask() {
         block(it)
         commit(variant.applicationId, it).execute()
     }
+
+    internal fun AndroidPublisher.Edits.updateTracks(
```

Comment:
This is only used in the `PublishApkTask`, why are we moving it to the base?

Reply:
That's there because the `modify` work will use it as well.

Reply:
That works for me, I'll refactor when I do the `modify` work.

Reply:
❓ Can't you do it now? Or at least move this back to where it was in the `PublishApkTask`? If you want, I'd be happy to do that for you. 😄